test/tall/regression/other/misc.unit

```diff
@@ -19,21 +19,4 @@ SomeVeryLongReturnType someFunctionName(
         })
         .then(________traverseDeps_depender__deps_);
   }
-}
->>> Split in long type parameter clause.
-abstract class StreamNotifierProviderBase<
-  NotifierT extends AsyncNotifierBase<T>,
-  T
-> extends ProviderWithNotifier<AsyncValue<T>, NotifierT>
-    with FutureModifier<T> {
-  // ...
-}
-<<<
-abstract class StreamNotifierProviderBase<
-  NotifierT extends AsyncNotifierBase<T>,
-  T
->
-    extends ProviderWithNotifier<AsyncValue<T>, NotifierT>
-    with FutureModifier<T> {
-  // ...
```

Comment:
I removed this from here because it's covered by 1348.unit. I found this snippet in a text file of mine and didn't realize it was already in an issue.